common/cmake/devices.cmake

```diff
@@ -929,6 +929,191 @@ function(ADD_FPGA_TARGET_BOARDS)
   endforeach()
 endfunction()
 
+function(ADD_BITSTREAM_TARGET)
+  set(options)
+  set(oneValueArgs NAME USE_FASM OUT_LOCAL_REL)
+  set(multiValueArgs INCLUDED_TARGETS)
+  cmake_parse_arguments(
+    ADD_BITSTREAM_TARGET
+    "${options}"
+    "${oneValueArgs}"
+    "${multiValueArgs}"
+    ${ARGN}
+  )
+
+
+  set(NAME ${ADD_BITSTREAM_TARGET_NAME})
+  set(INCLUDED_TARGETS ${ADD_BITSTREAM_TARGET_INCLUDED_TARGETS})
+  set(USE_FASM ${ADD_BITSTREAM_TARGET_USE_FASM})
+  set(OUT_LOCAL_REL ${ADD_BITSTREAM_TARGET_OUT_LOCAL_REL})
+
+  if("${USE_FASM}" STREQUAL "")
```

Comment:
Should USE_FASM just be an "options" arg if it doesn't take an argument?  I believe that's how you are using it here?